platform-api/src/main/java/io/datacater/core/stream/KafkaStreamsAdmin.java

```diff
@@ -173,9 +173,14 @@ public Map<String, String> getMetadata() {
    * Inspect (or retrieve) the most recent events of the Topic.
    *
    * @param limit Number of records to retrieve.
+   * @param distributed Mode of operation for the retrieval of records. - true: Distributed
```

Comment:
Instead of introducing the boolean flag `distributed`, I suggest introducing a more flexible parameter (or config option) that is defined as a `String` and that can point to an arbitrary number of sampling methods. Maybe call it `sampleApproach` or `sampleMethod`? This would allow us to introduce more sampling approaches later on and make the `inspect` method more flexible in the long term.

Reply:
Beyond that, to me, `distributed` is a bit confusing as a name for this approach because I immediately think of a distributed algorithm when reading it (maybe it's only me, though?). What about using a different name, like `uniform`? We then need a name for the other approach, too :)

Reply:
thanks for the feedback. i've renamed the variables to uniform and sequenced. feel free to suggest other namings 🙂

Reply:
Sounds good to me